doc/classes/PropertyTweener.xml

```diff
@@ -59,7 +78,25 @@
 
 				func tween_curve(v):
 				    return curve.sample_baked(v)
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				[Export]
+				public Curve Curve { get; set;}
```

Comment:
```suggestion
				public Curve Curve { get; set; }
```

Reply:
I have made changes that add these suggestions 👍